docs/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!--- If applicable, issue number goes here, e.g. GODRIVER-ABCD -->
+
+## Description
+<!--- A brief description of the pull request. -->
+
```

Comment:
Consider adding a "Rationale" section. It may not be obvious why an external contributor wants a change. My personal PR template is:

```
# Summary

# Background & Motivation
```